integrations/src/test/resources/io/jenkins/plugins/casc/GlobalToolGlobalMavenTest.yaml

```diff
@@ -0,0 +1,18 @@
+tool:
+  maven:
+    installations:
+      - name: "custom"
+        properties:
+          - installSource:
+              installers:
+                - maven:
+                    id: "3.6.3"
+  # TODO when https://github.com/jenkinsci/jenkins/pull/5008 is merged and released
```

Comment:
you can add an incrementals dependency in jenkins.version 

https://repo.jenkins-ci.org/incrementals/org/jenkins-ci/main/cli/2.263-rc30473.618ccc39c1b0/

`2.263-rc30473.618ccc39c1b0`